File: optimize/java-train/test/src/train/T0Test.java
package train;

import static org.junit.jupiter.api.Assertions.*;

import org.junit.jupiter.api.Test;

class T0Test {
    @Test
    void a() {
        assertEquals(0 * 11 + 3, T0.value());
    }

    @Test
    void b() {
        assertTrue(T0.label().startsWith("T0-"));
    }

    @Test
    void c() {
        assertEquals(T0.value(), T0.value());
    }

    @Test
    void d() {
        assertNotNull(T0.label());
    }

    @Test
    void e() {
        assertTrue(T0.value() >= 0);
    }
}

File: optimize/kotlin-train/test/src/T0Test.kt
import org.junit.jupiter.api.Assertions.*
import org.junit.jupiter.api.Test

class T0Test {
    @Test
    fun a() {
        assertEquals(0 * 11 + 3, t0Value())
    }

    @Test
    fun b() {
        assertTrue(t0Label().startsWith("T0-"))
    }

    @Test
    fun c() {
        assertEquals(t0Value(), t0Value())
    }

    @Test
    fun d() {
        assertNotNull(t0Label())
    }

    @Test
    fun e() {
        assertTrue(t0Value() >= 0)
    }
}
